alpha/apps/kaltura/modules/extwidget/actions/playManifestAction.class.php

```diff
@@ -1528,4 +1513,20 @@ protected function getScheduleTime()
 		$time += intval($this->getRequestParameter(kSimuliveUtils::SCHEDULE_TIME_OFFSET_URL_PARAM, 0));
 		return $time;
     }
+
+	protected function initEventData($event)
+	{
+		$offset = $this->getRequestParameter(kSimuliveUtils::SCHEDULE_TIME_OFFSET_URL_PARAM, "0"); // offset in sec
+		if ($offset)
+		{
+			$this->deliveryAttributes->setUrlParams('/' . kSimuliveUtils::SCHEDULE_TIME_OFFSET_URL_PARAM . '/' . $offset);
+		}
+		$this->entryId = $event->getSourceEntryId();
+		$sourceEntry = kSimuliveUtils::getSourceEntry($event);
+		$this->entry = $sourceEntry ? $sourceEntry : $this->entry;
+		if ($sourceEntry->getType() === entryType::MEDIA_CLIP && !$this->deliveryAttributes->getDeliveryProfileIds())
```

Comment:
if you checked if sourceEntry exist at line 1526, how can you access the object, if he does not exist?